bundles/net.tourbook/src/net/tourbook/ui/tourChart/TourChart.java

```diff
@@ -3667,6 +3775,41 @@ public void partIsVisible() {
       _valuePointTooltip.setShellVisible(true);
    }
 
+   private void removeActions_10_GraphActionsCustomTracks() {
```

Comment:
Why is there a sequence number _10_ in the method name when there are no surrounding other methods with a sequence number. It looks like a copy/paste fault without checking all.